packages/template-retail-react-app/package.json

```diff
@@ -71,6 +71,7 @@
         "save-credentials": "pwa-kit-dev save-credentials",
         "start": "cross-env NODE_ICU_DATA=node_modules/full-icu pwa-kit-dev start",
         "start:inspect": "npm run start -- --inspect",
+        "start:no-hmr": "npm run start -- --noHMR",
```

Comment:
I'm not sure if we want to have this as an project OOTB npm script. because if this feature works well, i'd imagine developers should never turn if off, or on a very very very rare case.